libs/barista-components/slider/src/slider.spec.ts

```diff
@@ -493,8 +493,7 @@ describe('DtSlider', () => {
       } = getElements(fixture);
       const { slider } = testComponent;
 
-      //<any, any> is needed because _updateSlider is private
-      const update = jest.spyOn<any, any>(slider, '_updateSlider');
+      const update = jest.spyOn(slider, '_updateSlider');
```

Comment:
This lead to errors when trying to build with Bazel

Reply:
We should fix this before we can merge this